packages/shared/src/components/plus/PlusComparingCards.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { FC } from 'react';
+import {
+  PaymentContextData,
+  ProductOption,
+} from '../../contexts/PaymentContext';
+import {
+  Typography,
+  TypographyColor,
+  TypographyTag,
+  TypographyType,
+} from '../typography/Typography';
+import { DevPlusIcon } from '../icons';
+import { Button, ButtonVariant } from '../buttons/Button';
+import { plusUrl } from '../../lib/constants';
+import { defaultFeatureList, plusFeatureList, PlusList } from './PlusList';
+import { IconSize } from '../Icon';
+
+export enum OnboardingPlans {
+  Free = 'Free',
+  Plus = 'Plus',
+}
+
+interface PlusCardProps {
+  currency: string;
+  onClickNext: () => void;
+  productOption?: ProductOption;
+}
+
+const PlusCard: FC<PlusCardProps> = ({
+  currency,
+  productOption: plan,
+  onClickNext,
+}) => {
+  const isPaidPlan = !!plan;
+  const price = plan?.price ?? '0';
+  const billingLabel = isPaidPlan ? `Billed ${plan?.label}` : 'Free forever';
+
+  return (
+    <div className="mx-auto w-70 max-w-full rounded-16 border border-border-subtlest-tertiary bg-surface-float p-4">
+      <div className="flex items-start justify-between gap-6">
+        <Typography
+          bold
+          className="mb-1.5"
+          tag={TypographyTag.H2}
+          type={TypographyType.Title3}
+          color={isPaidPlan ? TypographyColor.Plus : TypographyColor.Primary}
+        >
+          {isPaidPlan ? 'Plus' : 'Free'}
+        </Typography>
+        {isPaidPlan && (
+          <Typography
+            className="grid aspect-square h-8 w-8 place-content-center rounded-8 bg-surface-float"
+            tag={TypographyTag.Span}
+            color={TypographyColor.Plus}
+          >
+            <DevPlusIcon />
+          </Typography>
+        )}
+      </div>
+      <div>
+        <Typography bold tag={TypographyTag.Span} type={TypographyType.Title1}>
+          {!isPaidPlan && currency}
+          {price}
+        </Typography>
+        <Typography
+          color={TypographyColor.Tertiary}
+          type={TypographyType.Footnote}
+        >
+          {billingLabel}
+        </Typography>
+      </div>
+      {!isPaidPlan ? (
+        <Button
+          className="my-4 block w-full"
+          onClick={onClickNext}
+          title="Continue without Plus"
+          variant={ButtonVariant.Secondary}
+          type="button"
+        >
+          Continue
+        </Button>
+      ) : (
+        <Button
+          className="my-4 block w-full"
+          href={plusUrl}
+          tag="a"
+          target="_blank"
```

Comment:
Please add `rel={anchorDefaultRel}`

Reply:
I also think this should include the chosen option already right?

Reply:
Updated ✅